pkg/cmd/run/view/view.go

```diff
@@ -86,6 +93,10 @@ func NewCmdView(f *cmdutil.Factory, runF func(*ViewOptions) error) *cobra.Comman
 				}
 			}
 
+			if opts.Web && opts.Log {
+				return &cmdutil.FlagError{Err: errors.New("only one of --web or --log can be passed at a time")}
```

Comment:
I don't have an issue with this language but I think "specify only one of `--log` or `--web`"  would be more consistent with other error messages we give for mutually exclusive flags.